nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -42,6 +42,8 @@ In addition to numerous new and updated packages, this release has the following
 
 - [KDE Plasma](https://kde.org/de/plasma-desktop/) has been updated to version 5.27, see [the release notes](https://kde.org/announcements/plasma/5/5.27.0/) for what was changed.
 
+- [OLIMEX Teres-A64](https://www.olimex.com/Products/DIY-Laptop) received package declaration for uboot.
```

Comment:
We don't usually do relnotes for package additions.

Reply:
> We don't usually do relnotes for package additions. -- @K900 (https://github.com/NixOS/nixpkgs/pull/240827#discussion_r1248474041)

Should i remove it?

Reply:
It should definitely be moved to the 23.11 section at least. Ping @RaitoBezarius as release manager.